### PR TITLE
257 add nmf code

### PR DESCRIPTION
**Pull request by thanasions**
re-opened the pull request as after the rebase it was hard to follow what changed!

### CODE REVIEW

File: scripts/nmf_wrapper.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    calculate_term_weights = False
+    term_weights_to_sum = 10  # 0 to sum all weights
+    term_weights_to_print = 50
+
+    # run NMF on TFIDF
+    nmf = NMF(n_components=n_nmf_topics, random_state=1, alpha=.1, l1_ratio=.5, init='nndsvd'). \
+        fit(tfidf_mat)
+
+    # calculate feature/term 'scores'
+    if calculate_term_weights:
```
Comment (IanGrimstead) [inline scripts/nmf_wrapper.py]: This is always skipped? `calculate_term_weights = False` above...

Comment (user624086) [reply scripts/nmf_wrapper.py]: This is an extra bit of functionality. I can change to: calculate_term_weights = True.
